paimon-flink/paimon-flink-common/src/test/java/org/apache/paimon/flink/ReadWriteTableITCase.java

```diff
@@ -1213,10 +1213,10 @@ public void testInferParallelism() throws Exception {
                                         new HashMap<String, String>() {
                                             {
                                                 put(INFER_SCAN_PARALLELISM.key(), "true");
-                                                put(SCAN_PARALLELISM.key(), "-1");
```

Comment:
Perhaps we should keep this case and assert that it will throw an `IllegalArgumentException`. But `-1` should be changed to another illegal value (such as `-2`), as explained in the previous comment.